examples/streetlights-operation-security.yml

```diff
@@ -91,10 +107,10 @@ channels:
       message:
         $ref: '#/components/messages/dimLight'
       security:
-      # This operation level security implies the ability to send a message to 
-      # `smartylighting.streetlights.1.0.action.{streetlightId}.turn.on` with Authorization headers 
-      # with `streetlights:read` scope. It is also possible for the same channel when using `test_auth` 
-      # server to instead use credentials for security (scramSha256 in this example) specified on the server level
+      # This operation level security implies the ability to subscribe to messages from
+      # `smartylighting.streetlights.1.0.action.{streetlightId}.dim` channel with Authorization headers 
+      # that have `streetlights:read` scope. Note that an operation level security must still satisfy 
+      # security options specified at the server level.
        streetlights_auth:
          - streetlights:read  
```

Comment:
Security requirement object should be in the form of array, not object.

Reply:
Fixed.